MaterialShowcase/MaterialShowcase.swift

```diff
@@ -300,7 +300,8 @@ extension MaterialShowcase {
   /// Create a copy view of target view
   /// It helps us not to affect the original target view
   private func addTarget(at center: CGPoint) {
-    targetCopyView = targetView.copyView()
+//    targetCopyView = targetView.copyView()
+    targetCopyView = targetView.snapshotView(afterScreenUpdates: true)
```

Comment:
If using `snapshotView` API, it seems that we can not set custom tint color via `showcase.targetTintColor`. you might help me to confirm that.